<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Pornhub</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="imageLabel">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Pictures/Camera Roll/WIN_20160324_184651.JPG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="actFile">
        <property name="text">
         <string>Select File</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Ne pas appuyer ici</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="actProcess">
        <property name="text">
         <string>Process</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFiltres">
    <property name="title">
     <string>Filtres</string>
    </property>
    <widget class="QMenu" name="menuGaussien">
     <property name="title">
      <string>Gaussien</string>
     </property>
     <addaction name="actionAppliquer"/>
     <addaction name="separator"/>
     <addaction name="actionOptions"/>
    </widget>
    <addaction name="menuGaussien"/>
    <addaction name="actionCanny"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuFiltres"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCanny">
   <property name="text">
    <string>Canny</string>
   </property>
  </action>
  <action name="actionAppliquer">
   <property name="text">
    <string>Appliquer</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
